src/builtins/BuiltinTemporal.cpp

```diff
@@ -76,6 +76,28 @@ static void checkTemporalDuration(ExecutionState& state, const Value& value)
     }
 }
 
+static void checkTemporalInstant(ExecutionState& state, const Value& value)
+{
+    if (!(value.isObject() && value.asObject()->isTemporalInstantObject())) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, "wrong type");
+    }
+}
+
+static Value temporalInstantFromEpoch(ExecutionState& state, unsigned int argc, Value* argv, BigInt* epoch)
```

Comment:
Please change the type of argc to `size_t`

Reply:
I think the argc type is still not size_t.